### DISCUSSION TITLE
T

### DISCUSSION THREAD

**Discussion by amitmiglani91** [strimzi/strimzi-kafka-operator#8089] [General]
Upgrading Kafka crd version of 0.32 :- 

Strimzi operator is updated already 0.32 

> Kubectl apply -f myfile.yaml -n kafka 
getting this error :-

Warning: resource customresourcedefinitions/kafkas.kafka.strimzi.io is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
The CustomResourceDefinition "kafkas.kafka.strimzi.io" is invalid: status.storedVersions[0]: Invalid value: "v1beta1": must appear in spec.versions

Can you Please helm what is this issue

**Comment by scholzj** [comment]
That suggests you are upgrading from Strimzi 0.22 or earlier and you did not converted the CRDs. Please follow the docs for that -> there are several important steps such as the CRD conversion. So you should read that and follow that.

**Reply by scholzj** [reply]
PS: Fixing the title of the discussion would be also good ;-)

**Reply by amitmiglani91** [reply]
Sure, Next time will keep in mind. :)
Ok will read that. thanx..
